src/pyrtma/context.py

```diff
@@ -13,18 +13,46 @@ class RTMAContext:
     constants: Dict[str, Union[int, float, str]] = field(default_factory=dict)
     typedefs: Dict[str, Any] = field(default_factory=dict)
     MID: Dict[str, int] = field(default_factory=dict)
+    MON: Dict[int, str] = field(default_factory=dict)
     SDF: Dict[str, Type[MessageBase]] = field(default_factory=dict)
     MT: Dict[str, int] = field(default_factory=dict)
+    MTN: Dict[int, str] = field(default_factory=dict)
     MDF: Dict[str, Type[MessageData]] = field(default_factory=dict)
 
     def _clear(self):
         self.constants.clear()
         self.typedefs.clear()
         self.MID.clear()
+        self.MON.clear()
         self.SDF.clear()
         self.MT.clear()
+        self.MTN.clear()
         self.MDF.clear()
 
+    def message_name_from_id(self, message_id: int) -> Union[str, None]:
+        if message_id in self.MTN.keys():
+            return self.MTN[message_id]
+        else:
+            return None
```

Comment:
You should be able to accomplish the same thing for each of these wrappers in one line using [`dict.get()`](https://docs.python.org/3/library/stdtypes.html#dict.get)

```suggestion
    def message_name_from_id(self, message_id: int) -> Union[str, None]:
        return self.MTN.get(message_id)
```

Reply:
I'll update that throughout!